src/pages/management/configuration-api/index.mdx

```diff
@@ -4075,4 +4075,4 @@ Informs that a user has seen events up to a specific time.
 
 # Contact us
 
-If you found a bug or a typo, you can let us know directly on GitHub. In case of any questions or feedback, don't hesitate to contact us at [developers@livechat.com] (mailto:developers@livechat.com). We'll be happy to hear from you!
+If you found a bug or a typo, you can let us know directly on GitHub. In case of any questions or feedback, don't hesitate to contact us at developers@livechat.com. We'll be happy to hear from you!
```

Comment:
Cant we just use `<a href="mailto:developers@livechat.com">developers@livechat.com</a>` ?